apps/prairielearn/src/pages/administratorCourses/administratorCourses.js

```diff
@@ -19,8 +21,18 @@ router.get(
   '/',
   asyncHandler(async (req, res) => {
     res.locals.coursesRoot = config.coursesRoot;
-    const result = await sqldb.queryOneRowAsync(sql.select, []);
-    _.assign(res.locals, result.rows[0]);
+    const { institutions, course_requests } = await getCourseRequests(false);
+    const courses = await sqldb.queryRow(
+      sql.select,
```

Comment:
As long as we're here, consider giving this a better name.

Reply:
Updated name to `select_courses` in commit [46a7046](https://github.com/PrairieLearn/PrairieLearn/pull/9238/commits/46a7046a6311dae7797f62cb4227fcbf1a3ead90).